vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -139,15 +139,19 @@ func (a *APIInstaller) getResourceKind(path string, storage rest.Storage) (schem
 	}
 
 	object := storage.New()
-	fqKinds, _, err := a.group.Typer.ObjectKinds(object)
+	fqKinds, unversioned, err := a.group.Typer.ObjectKinds(object)
 	if err != nil {
 		return schema.GroupVersionKind{}, err
 	}
 
 	// a given go type can have multiple potential fully qualified kinds.  Find the one that corresponds with the group
 	// we're trying to register here
 	fqKindToRegister := schema.GroupVersionKind{}
+	unversionedCandidate := schema.GroupVersionKind{}
 	for _, fqKind := range fqKinds {
+		if unversioned && fqKind.Group == "" {
```

Comment:
Merged https://github.com/sttts/origin/pull/75. It reverts this patch in `APIInstaller. getResourceKind` and replaces the `Status` type by `Build` in the webhook storage.